wire-schema/src/commonMain/kotlin/com/squareup/wire/schema/MessageType.kt

```diff
@@ -50,6 +50,9 @@ data class MessageType(
 ) : Type() {
   private var deprecated: Any? = null
 
+  val isExtensible: Boolean
+    get() = extensionsList.isNotEmpty()
```

Comment:
What does having extensions make it extensible?

Reply:
This is the list of reservations for extending a message, no?

If there are reservations then you can do `extend Foo` or whatever in proto land. 

That's what this is referring to.